hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/TestHoodieSparkSqlWriter.scala

```diff
@@ -1158,7 +1172,9 @@ class TestHoodieSparkSqlWriter {
     val options = Map(
       DataSourceWriteOptions.RECORDKEY_FIELD.key -> "id",
       DataSourceWriteOptions.PRECOMBINE_FIELD.key -> "ts",
-      DataSourceWriteOptions.PARTITIONPATH_FIELD.key -> "dt"
+      DataSourceWriteOptions.PARTITIONPATH_FIELD.key -> "dt",
+      HoodieWriteConfig.INSERT_PARALLELISM_VALUE.key -> "2",
+      HoodieWriteConfig.UPSERT_PARALLELISM_VALUE.key -> "2"
```

Comment:
probably we can remove it completely from tests. our source code should deduce automatically

Reply:
Makes sense.  Fixed.